one-click-integration.sh

```diff
@@ -176,7 +176,7 @@ echo ""
 ## Decode response and save as a certificate file
 echo "Decoding and saving certificate"
 echo ""
-crt=$( echo ${crtb64} | base64 --decode > generated.crt )
+crt=$(echo "${crtb64}" | openssl enc -base64 -d > generated.crt )
```

Comment:
On my machine this only returns last line of the certificate (`-----END CERTIFICATE-----`), does this solution works on linux as well? Adding one-line processing made it for me.
```suggestion
crt=$(echo "${crtb64}" | openssl enc -base64 -d -A > generated.crt )
```

Reply:
Seems to work fine with Linux and Alpine openssl